library/Ivoz/Provider/Domain/Model/Feature/Feature.php

```diff
@@ -25,6 +25,22 @@ class Feature extends FeatureAbstract implements FeatureInterface
     public const RETAIL            = 11;
     public const VPBX              = 12;
 
+    /**
+     * Available features constants
+     */
+    public const QUEUES_IDEN = 'queues';
```

Comment:
I would remove every _IDEN

Reply:
I cannot do that because of this 
![image](https://github.com/irontec/ivozprovider/assets/101095813/497ccdb4-3bc3-4076-985c-f33bd172f922)

Reply:
nah, don't bother. Just deploy it